### ISSUE TITLE
Bug with a map due to anti-pause

### ISSUE THREAD

**Issue by receiver1** [Whitetigerswt/gtasa_crashfix#68]
Hello, I would like to discuss a bug with a map. When installing the latest pre-release version, an anti-pause causes a bug with the map. The fact is that the game does not stop during the opening of the map and constantly updates it. GTA thinks that they did not open the menu and did not render the rest of the map, with the exception of the square in which you are located. I ask you to add an item to the config to disable anti-pause.

**Comment by gabmzzn** [comment]
Same issue here, I was about to make the same post

**Comment by Whitetigerswt** [comment]
Will look into this

**Comment by uBlock-user** [comment]
> I ask you to add an item to the config to disable anti-pause.

Yes, a setting such as `antipause 0` in the `.cfg` which would indicate to disable anti-pause would work.
